control/timeresp.py

```diff
@@ -210,6 +210,9 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
 
     T : array_like, optional for discrete LTI `sys`
         Time steps at which the input is defined; values must be evenly spaced.
+        If None, `U` must be given and and `len(U)` time steps of sys.dt are
+        simulated. If sys.dt is None or True (undetermined time step), a dt
+        of 1.0 is assumed.
```

Comment:
suggetsed additional sentence:
"That is, if the system's timebase is not specified (`sys.dt` is `None`), then it is interpreted as a continuous-time system if `T` is provided, and a discrete-time system with `dt = 1` if it is not."
(may be worth a sanity check -- is this a good idea?)

Reply:
It seems reasonable that for an LTI with unspecified time base, if there is no `T` vector then the system is interpreted as discrete time (with inputs at each time given by `U`).  I suspect this will be a corner case in terms of usage and may not be something we want to encourage.  I would probably leave the docstring as is.

Reply:
> I would probably leave the docstring as is.

"As is in the PR" or "as is in current master"?

Reply:
As in the PR.